HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -3609,6 +3615,134 @@ def test_batteries
+    _test_default_vehicle_values(default_hpxml_bldg.vehicles[0], default_hpxml_bldg.ev_chargers[0], HPXML::BatteryTypeLithiumIon, 45.0, nil, 34.0, nil, 5000, 10, 0.18, HPXML::UnitsKwhPerMile, 0.75, ConstantDaySchedule, ConstantDaySchedule, ConstantMonthSchedule, ConstantDaySchedule, ConstantDaySchedule, ConstantMonthSchedule, 1600, HPXML::LocationOutside)
+
+    # Test w/ Ah instead of kWh
+    hpxml_bldg.vehicles[0].nominal_capacity_kwh = nil
+    hpxml_bldg.vehicles[0].nominal_capacity_ah = 987.0
+    hpxml_bldg.vehicles[0].usable_capacity_kwh = nil
+    hpxml_bldg.vehicles[0].usable_capacity_ah = 876.0
+    XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+    _default_hpxml, default_hpxml_bldg = _test_measure()
+    _test_default_vehicle_values(default_hpxml_bldg.vehicles[0], default_hpxml_bldg.ev_chargers[0], HPXML::BatteryTypeLithiumIon, nil, 987.0, nil, 876.0, 5000, 10, 0.18, HPXML::UnitsKwhPerMile, 0.75, ConstantDaySchedule, ConstantDaySchedule, ConstantMonthSchedule, ConstantDaySchedule, ConstantDaySchedule, ConstantMonthSchedule, 1600, HPXML::LocationOutside)
+
+    # Test defaults
+    hpxml, hpxml_bldg = _create_hpxml('base-vehicle-ev-charger.xml')
+    hpxml_bldg.vehicles[0].nominal_capacity_kwh = nil
+    hpxml_bldg.vehicles[0].nominal_capacity_ah = nil
+    hpxml_bldg.vehicles[0].usable_capacity_kwh = nil
+    hpxml_bldg.vehicles[0].usable_capacity_ah = nil
+    hpxml_bldg.vehicles[0].miles_per_year = nil
+    hpxml_bldg.vehicles[0].hours_per_week = nil
+    hpxml_bldg.vehicles[0].fuel_economy = nil
+    hpxml_bldg.vehicles[0].fuel_economy_units = nil
+    hpxml_bldg.vehicles[0].fraction_charged_home = nil
+    hpxml_bldg.vehicles[0].ev_charging_weekday_fractions = nil
+    hpxml_bldg.vehicles[0].ev_charging_weekend_fractions = nil
+    hpxml_bldg.vehicles[0].ev_charging_monthly_multipliers = nil
+    hpxml_bldg.vehicles[0].ev_discharging_weekday_fractions = nil
+    hpxml_bldg.vehicles[0].ev_discharging_weekend_fractions = nil
+    hpxml_bldg.vehicles[0].ev_discharging_monthly_multipliers = nil
+    hpxml_bldg.ev_chargers[0].location = nil
+    hpxml_bldg.ev_chargers[0].charging_power = nil
+    default_wkdy_sch_charge = '1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0'
+    default_wknd_sch_charge = '1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0'
+    default_mth_mult_charge = '1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0'
+
+    default_wkdy_sch_discharge = '0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.4611, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.4201, 0.4231, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0'
+    default_wknd_sch_discharge = '0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.3946, 0.0, 0.0, 0.0, 0.0, 0.3897, 0.3991, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0'
+    default_mth_mult_discharge = '1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0'
```

Comment:
You shouldn't have to hardcode these values here. Look at what we do for other building components.